docs/changelog.md

```diff
@@ -26,6 +26,9 @@ permalink: /changelog/
 * **feature-findinpage**
    * Find in Page now emits facts
 
+* **feature-awesomebar**
```

Comment:
Wrong name.

Reply:
Which name should it be? Isn't it in `mozilla.components.feature.awesomebar.provider`?

Reply:
Ah! You're right! My bad, I got wrapped up thinking about storage layers :)